### PR TITLE
chore: silent healthcheck endpoints logs

### PR DESCRIPTION
**Pull request by arddluma**
## Description

- Silent logs for`/version` and `/healthcheck` routes

## Types of changes
What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Further comments (optional)

-

### CODE REVIEW

**Review by sarthakprp — APPROVED** [review]

File: packages/api/src/server.ts [hunk 1]
```diff
@@ -45,6 +45,12 @@ export class Server {
     }
 
     app.addHook("preHandler", (req, reply, done) => {
+      
+      if (req.url === "/version" || req.url === "/healthcheck") {
```
Comment (ch4r10t33r) [inline packages/api/src/server.ts:49]: Pls refactor code

```
if (req.url === "/version" || req.url === "/healthcheck" ) {
    // do nothing
} else if (req.method == "POST") {
   //existing code
} else {
   // existing code
}
done()

```

**Review by ch4r10t33r — CHANGES_REQUESTED** [review]
Have you tested this change locally?

File: packages/api/src/server.ts (hunk 1 above)
Comment (arddluma) [reply packages/api/src/server.ts:49]: yes @ch4r10t33r , tested and it is working fine. I don't see neither /version nor /healthcheck being logged.
image

Comment (ch4r10t33r) [reply packages/api/src/server.ts:49]: Please merge after testing.

**Review by ch4r10t33r — APPROVED** [review]
Please merge after testing the new change.

**Pull request merged**